package.json

```diff
@@ -20,7 +20,8 @@
     "integration:watch": "nodemon --exec \"npm run integration\" --watch src"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.4.0",
+    "@babel/plugin-transform-runtime": "^7.13.10",
```

Comment:
dev does not work:

![image](https://user-images.githubusercontent.com/503185/113550640-fac36d80-9625-11eb-8144-1b97c7f6d5f7.png)

Reply:
I have updated the packages